### DISCUSSION TITLE
Токен

### DISCUSSION THREAD

**Discussion by Gang14881** [schr1k/AnonChat#1] [Q&A]
Приветствую,я не могу понять куда вставить токен. И что нужно установить для полноценного работы бота(конфиг).Заранее спасибо

**Comment by schr1k** [comment]
Легче всего будет переписать все переменные вида config.<что-токапсом>
Но я могу скинуть структуру файла config.py

**Reply by schr1k** [reply]
На самом деле не думал что-то кто-то захочет воспользоваться моим кодом) Там очень много кода для работы платежки (payok).

**Reply by Gang14881** [reply]
Мне понравился твой код,как все реализовано,если не сложно можешь скинуть структуру config? Про заглавные буквы я понял,можем вообще ,если тебе удобно,связаться в тг

**Comment by schr1k** [comment]
добавил config.py

**Comment by Gang14881** [comment]
![image](https://user-images.githubusercontent.com/65944051/235762713-3da7c307-b51a-40de-a47a-83cfb826f1fa.jpeg)
А что с этим делать?

**Reply by schr1k** [reply]
А это SQL) нужно создать таблицы чтобы бот правильно работал. Повторюсь этот репозиторий не был создан чтобы его копировать или на нем учиться, тем более новичкам. Если сильно интересно могу скинуть SQL запросы для создания таблиц.

**Reply by schr1k** [reply]
Сам файл это лог куда выводятся ошибки. Ошибка дословно говорит что нет таблицы users

**Reply by Gang14881** [reply]
Если скинешь,буду благодарен за помощь

**Reply by schr1k** [reply]
Хороший курс по SQL на русском - https://stepik.org/course/Интерактивный-тренажер-по-SQL-63054/

**Reply by schr1k** [reply]
Описал в readme.md процесс установки. Если что-то непонятно или не работает обязательно пишите.
